dwave/optimization/mathematical.py

```diff
@@ -154,6 +156,41 @@ def concatenate(array_likes : typing.Union[collections.abc.Iterable, ArraySymbol
     raise TypeError("concatenate takes one or more ArraySymbol as input")
 
 
+def divide(x1: ArraySymbol, x2: ArraySymbol) -> Divide:
+    r"""Return an element-wise division on the given symbols.
+
+    In the underlying directed acyclic expression graph, produces a
+    ``Divide`` node if two array nodes are provided.
+
+    Args:
+        x1, x2: Input array symbol.
+
+    Returns:
+        A symbol that divides the given symbols element-wise.
+        Dividing two symbols returns a
+        :class:`~dwave.optimization.symbols.Divide`.
+
+    Examples:
+        This example divides two integer symbols.
+        Equivalently, you can use the ``/`` operator (e.g., :code:`i /j`).
+
+        >>> from dwave.optimization import Model
+        >>> from dwave.optimization.mathematical import divide
+        ...
+        >>> model = Model()
+        >>> i = model.integer(2)
+        >>> j = model.integer(2)
+        >>> k = divide(i, j)   # alternatively: k = i / j
```

Comment:
Probably missing something but won't this fail as `j`'s bounds will include 0 by default? Any way, might be good to include a note in this docstring about the divisor needing to be strictly negative or positive like you added in the release note.

Reply:
I am also surprised about that... Wonder if something is going wrong. If you put this example into the unittests does it raise like you'd expect?

Reply:
Ah, it needs to be added explicitly to https://github.com/dwavesystems/dwave-optimization/blob/main/docs/reference/math.rst

The doc builds are a weird mix of explicit/implicit. I bet there are others missing as well.

Reply:
I did and it does